templates/postgresql-server.json

```diff
@@ -8,16 +8,20 @@
         "description": "Name of the PostgreSQL server"
       }
     },
-    "postgresSku": {
-      "type": "object",
-      "defaultValue": {
-        "name": "GP_Gen5_2",
-        "tier": "GeneralPurpose",
-        "size": 5120,
-        "family": "Gen5",
-        "capacity": 2
-      }
-    },
+    "cpu" : {
+      "type": "int",
+      "defaultValue": 2
+  },
+  "postgresSku": {
+    "type": "object",
+    "defaultValue": {
+      "name": "GP_Gen5_2",
```

Comment:
```suggestion
      "name": "[concat('GP_Gen5_', parameters('dbServerCpuCores'))]",
```
or just

```suggestion
      "name": "GP_Gen5",
```

Reply:
see this:- https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-postgresql-server-database-using-arm-template?tabs=azure-portal

Reply:
ah okie..thought 2 referrred to the number of cores. ignore this one. thanks.

Reply:
actually https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-azure-cli#create-an-azure-database-for-postgresql-server